micro-rdk/src/common/entry.rs

```diff
@@ -105,8 +100,26 @@ pub async fn serve_web_inner<S>(
     // initialization is done
     let _ = Timer::after(std::time::Duration::from_millis(60)).await;
 
-    let (app_config, cfg_response, cfg_received_datetime) =
-        app_client.get_config(Some(network.get_ip())).await.unwrap();
+    let robot_credentials = app_client.robot_credentials();
+
+    let (cfg_response, cfg_received_datetime) = app_client
+        .get_app_config(Some(network.get_ip()))
+        .await
+        .unwrap();
+
+    let rpc_host = cfg_response
+        .config
+        .clone()
+        .unwrap_or(RobotConfig {
+            ..Default::default()
+        })
+        .cloud
+        .clone()
+        .unwrap_or(CloudConfig {
+            ..Default::default()
+        })
```

Comment:
is it so we know what's being unwrapped at each level that you're not using `unwrap_or_default()`?

Reply:
I tried, `Default` isn't implemented for those types